console/src/hardware/task/ontology.tsx

```diff
@@ -74,7 +72,7 @@ const TreeContextMenu: Ontology.TreeContextMenu = (props) => {
         });
         break;
       case "link": {
-        const url = `synnax://cluster/${clusterKey}/task/${selection.resources[0].id.key}`;
+        const url = `synnax://cluster/${client.key}/task/${selection.resources[0].id.key}`;
```

Comment:
Use `clusterKey` and not `client.key`

Reply:
This won't work. The client key is not equal to the cluster key. Use the cluster key instead.